src/core/public/ui_settings/__snapshots__/ui_settings_service.test.ts.snap

```diff
@@ -12,7 +12,7 @@ exports[`#start constructs UiSettingsClient and UiSettingsApi: UiSettingsApi arg
   ],
   "results": Array [
     Object {
-      "isThrow": false,
+      "type": "return",
```

Comment:
These snapshots look like they're testing more jest internals rather than just the call arguments themselves. Can we fix these too?

Reply:
Mocked functions are inside JS objects that serialised for snapshot testing. I didn't find a cheap way to fix it and decided to keep it as is.
As an alternative solution, we can define custom serialiser for mocked functions https://jestjs.io/docs/en/configuration#snapshotserializers-array-string

Reply:
Perfectly good reason.